contrib/nix/flake.nix

```diff
@@ -139,11 +188,13 @@
             src = fileSetForCrate ../..;
           }
         );
+
+        treefmt = treefmt-nix.lib.evalModule pkgs (import ./treefmt.nix);
```

Comment:
If possible I'd leave the formatting bit out so the PR does one thing only and does it well.